zsh-system-clipboard.zsh

```diff
@@ -40,11 +40,11 @@ if [[ "$ZSH_SYSTEM_CLIPBOARD_METHOD" == "" ]]; then
 			fi
 			;;
 		linux*|freebsd*)
-			if [[ "$DISPLAY" == "" ]]; then
+			if _zsh_system_clipboard_command_exists wl-copy && [[ "$WAYLAND_DISPLAY" != "" ]]; then
```

Comment:
Let's remove all `$DISPLAY` checks and change this line to:

```suggestion
			if _zsh_system_clipboard_command_exists wl-copy; then
```

Reply:
But you can have all of them installed at the same time. Then if you relogin in Xorg session it won't pick the right one.